models/eprop_readout.h

```diff
@@ -55,43 +55,82 @@ E-prop plasticity was originally introduced and implemented in TensorFlow in [1]
 The membrane voltage time course :math:`v_j^t` of the neuron :math:`j` is given by:
 
 .. math::
-    v_j^t &= \kappa v_j^{t-1}+\sum_{i \neq j}W_{ji}^\mathrm{out}z_i^{t-1}
-             -z_j^{t-1}v_\mathrm{th} \,, \\
-    \kappa &= e^{-\frac{\Delta t}{\tau_\mathrm{m}}} \,,
+  v_j^t &= \kappa v_j^{t-1}+ \zeta \sum_{i \neq j} W_{ji}^\text{out} z_i^{t-1} \,, \\
+  \kappa &= e^{ -\frac{ \Delta t }{ \tau_\text{m}} } \,, \\
+  \zeta &=
+    \begin{cases}
+      1 \\
+      1 - \alpha
+    \end{cases} \,, \\
 
-whereby :math:`W_{ji}^\mathrm{out}` are the output synaptic weights and
-:math:`z_i^{t-1}` are the recurrent presynaptic spike state variables.
+whereby :math:`W_{ji}^\text{out}` is the output synaptic weight matrix and
+:math:`z_i^{t-1}` is the recurrent presynaptic spike state variable.
 
 Descriptions of further parameters and variables can be found in the table below.
 
-An additional state variable and the corresponding differential
-equation represents a piecewise constant external current.
+The spike state variable of a presynaptic neuron is expressed by a Heaviside function:
 
-See the documentation on the ``iaf_psc_delta`` neuron model for more information
-on the integration of the subthreshold dynamics.
+.. math::
+  z_i^t = H \left( v_i^t - v_\text{th} \right) \,. \\
+
+An additional state variable and the corresponding differential equation
+represents a piecewise constant external current.
+
+See the documentation on the :doc:`iaf_psc_delta<../models/iaf_psc_delta/>` neuron model
+for more information on the integration of the subthreshold dynamics.
 
-The change of the synaptic weight is calculated from the gradient :math:`g` of
-the loss :math:`E` with respect to the synaptic weight :math:`W_{ji}`:
-The change of the synaptic weight is calculated from the gradient
-:math:`\frac{\mathrm{d}{E}}{\mathrm{d}{W_{ij}}}=g`
+The change of the synaptic weight is calculated from the gradient :math:`g^t` of
+the loss :math:`E^t` with respect to the synaptic weight :math:`W_{ji}`:
+:math:`\frac{\text{d}{E^t}}{\text{d}{W_{ij}}} = g^t`
 which depends on the presynaptic
 spikes :math:`z_i^{t-1}` and the learning signal :math:`L_j^t` emitted by the readout
 neurons.
 
+In between two presynaptic spikes, the gradient is calculated for each time step until the cutoff time point, i.e., for
+:math:`t \, \epsilon \, \left[ t_\text{s}^{t-1}, \text{min} \left( t_\text{s}^{t-1} + {\Delta t}_\text{c},
+t_\text{s}^t \right) \right]`:
+
```

Comment:
```suggestion
In the interval between two presynaptic spikes, the gradient is calculated
at each time step until the cutoff time point. This computation occurs over
the time range:
:math:`t \in \left[ t_\text{prev}, \min \left( t_\text{prev} + \Delta t_\text{c}, t_\text{curr} \right) \right]`.
Here, :math:`t_\text{prev}` represents the time of the previous spike that
stimulated the synapse, while :math:`t_\text{curr}` is the time of the
current spike, which triggers the application of the learning rule and the
subsequent synaptic weight update. The cutoff :math:`\Delta t_\text{c}` 
defines the maximum allowable interval for integration between spikes. 
The expression for the gradient is given by
```
The variables $t_\text{s}^{t-1}$ and $t_\text{s}^{t}$ might be confusing because they include a variable named $t$ with a superscript $t$. I suggest renaming them to $t_\text{prev}$ and $t_\text{curr}$ for better clarity. Although the meanings of these variables can be inferred from the context,  I think it would be more precise and formal to explicitly state what they represent. I think also that $\in$ should be preferable over $\epsilon$.

Reply:
Fixed in https://github.com/JesusEV/nest-simulator/pull/16/commits/06a855a09cbb1963f3f4c1a49ac30baaa5ec4b73